projects/batfish-common-protocol/src/test/java/org/batfish/common/ContainerTest.java

```diff
@@ -1,39 +0,0 @@
-package org.batfish.common;
```

Comment:
Sorry, you are right. And now it still shows as a deletion and creation, it should because I copied the file and delete the origin one separately instead of using a 'git mv' command.